PROCESS.md

```diff
@@ -0,0 +1,163 @@
+# JSON Schema Specification Development Process
+This document formally defines the process used to evolve the JSON Schema
+specification as of the first stable release in 2023. It applies to the core
+specification and the specifications for any dialects and vocabularies
+maintained by the JSON Schema Org. It does not necessarily apply to everything
+the JSON Schema Org maintains. For example, media type registration follows the
+IETF process. Certain components used within JSON Schema, such as Relative JSON
+Pointer, may also follow a separate process.
+
+## Canonical URLs
+There MUST be a stable canonical URL for referencing any specification that
+follows this process. If the spec is made available from any other URLs, they
+SHOULD redirect to the canonical URL. If the canonical URL is changed in the
+future, all previous canonical URLs MUST remain accessible as redirects to the
+current URL.
+
+## Mutable Spec
+There is only one version of the specification and it can be updated at any time
+as long as those changes don't violate the compatibility rules identified by the
+stability model. Bug fixes, clarifications, and other non-functional spec
+updates can be merged at any time even for stable features. A change-log blog
+post SHOULD be prepared quarterly to give visibility to any changes that were
+merged in the last quarter.
+
+## The Stability Model
+Every feature in the spec has a release status. It's either evolving, stable, or
+deprecated. Flags are used to show the status a feature is in. If a feature
+doesn't have a flag, it's considered stable. If it has a year flag (such as
+`2024`) that's the year the feature reached stability. The other flags indicate
```

Comment:
If we're considering flagless features as stable, is there another indicator as to when the feature reached stability?  Or does "doesn't have a flag" only pertain to the status flag?  Do we expect all stable features to have a year flag?  If one doesn't is that just an admin error that needs to be fixed?

Maybe it's easier to require that features have flags so that if they don't it's something to be fixed.  Having an allowance for not having a flag seems to raise a lot of questions.

Reply:
A part of the spec can only be unflagged if it's a stable feature that was introduced in the original stable release.

My thinking was that it would be too noisy to flag everything in the spec. It would be hard to see `2024` flags in a sea of `2023` flags. The first stable release wouldn't have any `2023` flags because it's unambiguous when those features were introduced since `2023` is the first release. Stable flags would start to be added for `2024`. That means that the number of flags in the spec should be fairly small and therefore stand out to readers.

I'll revise to clarify the purpose of unflagged features, but let's continue the discussion and get some more opinions if you still dislike the idea.

Reply:
Maybe we keep the "unflagged implies stable" idea, and instead of flagging the year for future features, we have a `newly-stable` flag that identifies features that are new for the year's release.  Then the next year, those `newly-stable` flags are cleared from those features so that the new `newly-stable` features stand out.

Reply:
Your changes look fine.

Reply:
Let's discuss it at the next OCWM